frontend/apps/server/src/utils/stringTransformer.ts

```diff
@@ -0,0 +1,17 @@
+import { Buffer } from 'node:buffer';
+
+export const base64Url = {
+  encode: (plainText: string) => Buffer.from(plainText).toString('base64')
+    .replace(/\+/g, '-')
+    .replace(/\//g, '_')
+    .replace(/=+$/, ''),
+  decode: (encodedText: string) => {
+    while (encodedText.length % 4) {
+      encodedText += '=';
+    }
```

Comment:
p3; 4의 배수 만들어주기. 요런 유틸로 따로 뺄 수 있을 것 같습니다.
```suggestion
const padBase64 = (str: string): string => {
  const padLength = (4 - (str.length % 4)) % 4;
  return str + '='.repeat(padLength);
};
```